testkit-backend/lib/testkit/backend/messages/requests/check_driver_is_encrypted.rb

```diff
@@ -0,0 +1,9 @@
+module Testkit::Backend::Messages
+  module Requests
+    class CheckDriverIsEncrypted < Request
+      def process
+        named_entity('DriverIsEncrypted', encrypted: fetch(driverId).is_encrypted?)
```

Comment:
that's not compliant with https://github.com/jruby/jruby/wiki/CallingJavaFromJRuby#alternative-names-and-beans-convention